src/drivers/imx/ipc.c

```diff
@@ -294,3 +294,76 @@ int ipc_platform_poll_tx_host_msg(struct ipc_msg *msg)
 }
 
 #endif
+
+/*
+ * The _memmap_cacheattr_reset linker script variable has
```

Comment:
This looks like the wrong file, should really create a new file src/arch/xtensa/.../cache_attr.c

Reply:
I saw that, so far, only imx is using the _memmap_cacheattr_reset linker script variable to determine if an address is cacheable or not. 
But, you're right, other targets may come and use this attribute.
I was thinking that this might be a temporary solution until de gcc issue will be fixed.
But, I'll do the changes.

Reply:
I've added `cache_attr.c` in `src/arch/xtensa/drivers/`.